### DISCUSSION TITLE
Multiple Enviroments

### DISCUSSION THREAD

**Discussion by brocard** [vercel/community#122] [Help] [answered]
Hello community, we know actually we have 3 environments for every project at Vercel, **Development, Preview and Production**, its posible create a custom or extra environment something like `Staging`?

**Comment by wiiiimm** [comment, marked as answer]
@brocard you can set your own preview environments by mapping it to a custom git branch. Take a look at https://vercel.com/docs/concepts/projects/environment-variables#preview-environment-variables for details.

**Reply by brocard** [reply]
Thanks thanks x1000

**Comment by tomevans18** [comment]
Any idea how this would work with the new shared environment variables?
We have dev, staging and prod for all our apps, and it seems like we will still have to manage the staging envs per app which is a shame.
It feels like we need a proper setup for this in Vercel, meaning there would be:
- dev
- preview
- staging <-- NEW
- prod
